src/Strategy/Node.hs

```diff
@@ -88,7 +96,7 @@ discover dir = context "NodeJS" $ do
       pure []
     else do
       globalGraph <- context "Building global workspace graph" $ pure $ buildManifestGraph manifestMap
-      graphs <- context "Splitting global graph into chunks" $ fromMaybeText "Detected cyclic package.json graph" $ splitGraph globalGraph
+      graphs <- context "Splitting global graph into chunks" $ fromMaybe CyclicPackageJson $ splitGraph globalGraph
```

Comment:
[todo]:  I don't like how I laid this error out, I think I just forgot to refactor it after making some change.
```suggestion
      -- TODO: refactor splitGraph to report which cycle we hit, not just report some unknown cycle
      graphs <- context "Splitting global graph into chunks" $ fromMaybe CyclicPackageJson $ splitGraph globalGraph
```